### ISSUE TITLE
Spells cost double of their cost (BB #850)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#850]
This issue was migrated from bitbucket.
**Original Reporter:** Arnibou
**Original Date:** 24.04.2015 17:58:35 GMT+0000
**Original Priority:** critical
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850

<hr>

For example: Heroic strike must consume 15 rage while it consume 30 rage.
And same for all spells with energy, mana and rage.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Arnibou
**Original Date:** 24.04.2015 18:42:17 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850/#comment-17661589

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Arnibou
**Original Date:** 24.04.2015 18:43:24 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850/#comment-17661607

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Arnibou
**Original Date:** 24.04.2015 18:50:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850/#comment-17662126

<hr>

Cause by this commit: [d2a2313](https://bitbucket.org/oregon/oregoncore/commits/d2a2313d685b050726b6110abf893d6f556c0fbe)

Here probably:

```
#!c++

if (!m_IsTriggeredSpell)
     {
-        //TakePower();
+       TakePower();
         TakeReagents();                                         // we must remove reagents before HandleEffects to allow place crafted item in same slot
     }

```

Function TakePower() should not be correct

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 24.04.2015 19:51:24 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850/#comment-17663076

<hr>

You mean in  3d53299 ...

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 24.04.2015 19:51:50 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/850/#comment-17663082

<hr>

Fix #850

→ <<cset 4e3e38a74eb0>>
